lib/app-routes/common/auth/keys.js

```diff
@@ -40,6 +42,9 @@ exports.register = function(server, options, next) {
                     password: joi.binary().encoding('base64').required()
                 }
             },
+            response: {
+                schema: internals.credentialSchema
```

Comment:
I created a new user object. referenced it in response validation of Users and finallly exported it.